services/api/src/routes/auth.js

```diff
@@ -5,31 +5,51 @@ const { authenticate, fetchUser } = require('../utils/middleware/authenticate');
 const { createTemporaryToken, generateTokenId } = require('../utils/tokens');
 const { sendTemplatedMail } = require('../utils/mailer');
 const { User, Invite, AuditEntry } = require('../models');
+const config = require('@bedrockio/config');
 
 const mfa = require('../utils/mfa');
+const sms = require('../utils/sms');
+
 const { verifyLoginAttempts, verifyPassword } = require('../utils/auth');
 
 const router = new Router();
 
+const APP_NAME = config.get('APP_NAME');
+
+//TODO: reuse the FIXED_SCHEMA validator from the validation.js file?
+const phone = yd.string().custom(async (val) => {
```

Comment:
we can remove this as `validate: "phone"` will be a thing going forward. also this and other code is duplicated below it seems?

Reply:
what do i need to write here instead ?

```
.post(
    '/login/send-sms',
    validateBody({
      phoneNumber: phone.required(),
    }),
```

Reply:
Just remove that part and add to the model:

```json
  "phoneNumber": {
    "type": "String",
    "validate": "phone"
  }
```

Reply:
Oh nm I see this is a one-off. In that case just say `yd.string().phone()`!

Reply:
I am gonna let it sit until we remove the custom schema for the register endpoint